aten/src/ATen/native/cuda/BatchLinearAlgebra.cu

```diff
@@ -2710,14 +2710,12 @@ std::tuple<Tensor, Tensor, Tensor> _svd_helper_cuda_legacy(const Tensor& self, b
     U_working_copy.zero_();
```

Comment:
I don't think you need this, U_working_copy and VT_working_copy are empty in this case